admin-ui/plugins/auth-server/components/Agama/AgamaAliasListPage.js

```diff
@@ -61,14 +71,14 @@ function AliasesListPage() {
     dispatch(getJsonConfig({ action: {} }))
   }, [dispatch])
 
-  // Build actions based on permissions
   useEffect(() => {
+    const t = (s) => s
```

Comment:
Remove unnecessary translation function wrapper

Creating a local identity function `const t = (s) => s` that just returns the input string is unnecessary and confusing since there's already a real translation function from useTranslation.


```diff
-    const t = (s) => s
     const actions = []
```